model/shared/src/main/scala/explore/model/boopickle/CommonPicklers.scala

```diff
@@ -80,6 +81,9 @@ trait CommonPicklers {
       Enumerated[A].tag(_)
     )
 
+  def picklerNewType[A: Pickler](nt: NewType[A]): Pickler[nt.Type] =
```

Comment:
Can this be `given` so we don't have to manually define instances?

Reply:
We can't pass an explicit parameter to define a given (I think) maybe we'd need to make `NewType` a typeclass for that

Reply:
Oh you are right, we would need a type class, which defeats the purpose.

I'm guessing as an alternative we could add to `NewType`:
`given pickler(using p: Pickler[Wrapped]): Pickler[Type]        = p`

(I'm wondering if we should do the same with `Eq`)

Reply:
Good idea, let's try that

Reply:
The downside is that we make `NewType` depend on `Pickle`, which may be an issue if we want to move `NewType` to `lucuma-core`.